### PR TITLE
Feature/7776 refactor archive annotation

### PR DESCRIPTION
**Pull request by danielafeitosa**
Add Perma.cc as option on keep bot
Create new archiver annotation "archiver" to replace the specific archivers (archive_org, archive_is, perma_cc)
Change the archivers field instances to be optional
Add a migration and task to replace the specific archivers annotation:
  * There will be one of type "archiver"
  * On DynamicAnnotation::Field, replaced the annotation with the new annotation type 'archiver'

### CODE REVIEW

File: app/models/bot/keep.rb [hunk 1]
```diff
@@ -17,7 +17,8 @@ def self.archiver_annotation_types
       'keep_backup',    # VideoVault
```
Comment (infojunkie) [inline app/models/bot/keep.rb]: Is `keep_backup` still active on either Pender or Check API ?

File: app/models/bot/keep.rb [hunk 2]
```diff
@@ -17,7 +17,8 @@ def self.archiver_annotation_types
...
       'pender_archive', # Screenshot
```
Comment (infojunkie) [inline app/models/bot/keep.rb]: In a future refactoring, we would rename this to `screenshot`.

File: app/models/bot/keep.rb (hunk 1 above)
Comment (danielafeitosa) [reply app/models/bot/keep.rb]: It was removed from Pender. On CheckAPI it is not active now, there might be only the old annotations

Comment (infojunkie) [reply app/models/bot/keep.rb]: OK thanks will file the a refactoring to review and remove Video Vault archives.

**Pull request merged**